.teamcity/settings.kts

```diff
@@ -906,7 +906,7 @@ object RunCalypsoE2eDesktopTests : BuildType({
 
 				export LIVEBRANCHES=true
 				export NODE_CONFIG_ENV=test
-				#export MAGELLANDEBUG=true
+				export MAGELLANDEBUG=true
```

Comment:
I don't know enough about Magellan - is this something we need to keep?

Reply:
Yes. Without this Magellan hides the output of each individual `mocha` process, but we need that output because that's what tells TeamCity about the tests.

I'll add a comment because it is not obvious what this is doing.